tools/ldcontext_generator.py

```diff
@@ -173,6 +201,39 @@ def aggregate_ld_context(f, uri_prefix, predefined_mappings):
     for p in ld_context:
         aggregated_context[p] = ld_context[p]
 
+        # adding related specifications and schemas
+        if p not in terms_list['terms']:
+            terms_list['terms'][p] = {'specifications': list(),
+                                      'schemas': list()}
+
+        terms_list['terms'][p]['schemas'].append(schema_url.format(f.split('../')[1]))
+
+        file_to_add = find_file(f, terms_mappings)
+        if file_to_add:
+            terms_list['terms'][p]['specifications'].append(file_to_add)
+        else:
+            alert_list.append(f)
+
+
```

Comment:
can you add comment describing what this function does?

Reply:
Fixed in 8933b1ef27836a637fd56c3530e5b226b30a2fdb